lib/tasks/publishing_api.rake

```diff
@@ -2,6 +2,32 @@ require_relative "../publish_organisations_api_route"
 require_relative "../special_route_publisher"
 
 namespace :publishing_api do
+  desc "Publish coronavirus_landing_page to publishing api"
+  task publish_coronavirus_landing_page: :environment do
+    content_id = "774cee22-d896-44c1-a611-e3109cce8eae"
+    params = {
+      base_path: "/coronavirus",
+      publishing_app: "collections-publisher",
+      rendering_app: "collections",
+      public_updated_at: Time.zone.now.iso8601,
+      update_type: "minor",
+      schema_name: "edition",
```

Comment:
We don't need this line I think

Reply:
thanks removed

Reply:
@sihugh Getting `["Schema could not be validated as the schema_name was not provided"]` when running the task on integration https://deploy.integration.publishing.service.gov.uk/job/run-rake-task/132401/console I'll stick it back in!